dashboard/test/ui/features/star_labs/signin_callout.feature

```diff
@@ -37,7 +54,9 @@ Scenario: See age callout, not signin callout on hour of code
   And I wait until I don't see selector "#p5_loading"
   And I select age 10 in the age dialog
 
-Scenario: After I dismiss the callout, it should not reappear upon refresh
+# Cookies operate on mobile according to device settings: test only passes on rerun
+@no_mobile
```

Comment:
Unrelated to this change, but this reminds me, did we check that the callout looks and works ok on mobile?

Reply:
I've been testing on my iPhone, and it looks good!